apps/walletd/contexts/events/utils.ts

```diff
@@ -1,7 +1,28 @@
 import BigNumber from 'bignumber.js'
 import { WalletEvent } from '@siafoundation/walletd-types'
 
-export function calculateScOutflow(e: WalletEvent) {
+export function calculateScValue(e: WalletEvent) {
+  // TODO: how do we calculate the value of a v2 transaction?
+  if (e.type === 'v2Transaction') {
+    return undefined
```

Comment:
You can get the value of a v2 siacoin and siafund input from the `siacoinInputs[0].parent.siacoinOutput.value`